frontend/src/app/game/events-layout/events-layout.component.ts

```diff
@@ -15,12 +15,13 @@ export class EventsLayoutComponent {
   constructor(public gameService: GameService) {
   }
 
-  resumeEvent(eventMitigation?: EventMitigation) {
-    if (eventMitigation !== undefined) {
-      this.gameService.game.applyMitigationActions({eventMitigations: [eventMitigation]});
+  resumeEvent(event: Event, chosenMitigation: EventMitigation | undefined) {
```

Comment:
You can do this using optional (`?`) param, or a default value. In either case when invoked, `undefined` arg is not needed.